src/plugins/koops_event.conf

```diff
@@ -30,23 +30,23 @@ EVENT=post-create type=Kerneloops remote!=1
 #EVENT=report_Kerneloops type=Kerneloops
         reporter-kerneloops
 
-EVENT=report_Bugzilla type=Kerneloops
+EVENT=report_Bugzilla type=Kerneloops mce!=non-fatal mce!=fatal
         reporter-bugzilla -b \
                 -F /etc/libreport/plugins/bugzilla_format_kernel.conf
 
 # Send micro report
-EVENT=report_uReport type=Kerneloops
+EVENT=report_uReport type=Kerneloops mce!=non-fatal mce!=fatal
```

Comment:
Well, that is not OK. The last message would cause many administrators to start pondering what is wrong and be sure you would get a bug report sooner or later.

The fix is easy -> don't disable the "EVENT=notify", but add 'if [ -e mce ]; then ...; else ...; fi`. If you want to be really cool, you can print out an informative message - but don't forget that that message must not scary administrators - it must be something positive.